t/t5538-push-shallow.sh

```diff
@@ -124,4 +124,21 @@ EOF
 	git cat-file blob $(echo 1|git hash-object --stdin) >/dev/null
 	)
 '
+
+test_expect_success 'push new commit from shallow clone to origin is efficient' '
+	git init origin &&
+	echo a >origin/a &&
+	git -C origin add a &&
+	git -C origin commit -m "base" &&
```

Comment:
Not a big issue, but wouldn't this be easier written as `test_commit -C origin a`, and `test_commit -C origin b` instead of the next three lines?

Reply:
I always forget about that form of `git commit`. Thanks.

Reply:
(Unfortunately, that doesn't work because these files are untracked before the commit commands.)